common/openapi.yml

```diff
@@ -303,6 +303,7 @@ components:
       description: |
         ⚠️ Do not add sensitive data. All catalog entries are publicly accessible to all Terra users. ⚠️
       type: object
+      required: [storageSystem, storageSourceId]
```

Comment:
It seemed like this was an oversight, so I just wanted to driveby fix it. We had a test that was generating NPE for using nulls here, and I couldn't figure out any realistic case someone would want to create a catalog entry without a storage system or storage source id.